### DISCUSSION TITLE
Projects using MobX.dart

### DISCUSSION THREAD

**Discussion by pavanpodila** [mobxjs/mobx.dart#767] [Show and tell]
It's awesome to see a community building up around MobX for the Dart and Flutter platforms. 🚀

If you can share about your project, please do comment and let us all know. I am sure it's a diverse world out there. We would love to see all the places where it is being used 🙏. 

A section on [mobx.pub](https://mobx.pub) will be created too.

**Comment by pavanpodila** [comment]
📷 **[SeizeYourSpace](https://github.com/pavanpodila/SeizeYourSpace)**: a fun app to capture candidate details at a conference booth, with a Selfie 🤳

**Comment by patelmanjeet** [comment]
Need some more example with CRUD + Rest API + Providers (MultiProvider) + Pagination/Infinite scroll data + Organizing Stores

**Comment by coyksdev** [comment]
[ExpansionPanel](https://github.com/campanagerald/mobx_expansion_panel_list_example): Sample of using Mobx in ExpansionPanel. Open for comments and suggestions. Thanks!

**Comment by MaikuB** [comment]
I've done a simple hacker news one that uses provider and loads more stories upon scrolling [here](https://github.com/MaikuB/supnews)

Note: i'm aware there's an existing example but I haven't seen one making use of provider and pagination that would be similar to what @patelmanjeet is asking about

**Comment by taliptako** [comment]
[UniQuote](https://play.google.com/store/apps/details?id=net.uniquote.uniquote_flutter) A Quote Database that you can create your favorite list and follow specific tags or users

**Comment by amondnet** [comment]
[Balance Friends (밸런스프렌즈)](https://apps.apple.com/app/id1466899980) is a chatbot app (500000 download +)  to help your diet. Currently only Korean is supported.

**Comment by bbedward** [comment]
[Blaise](https://blaisewallet.com) - a wallet for the PascalCoin cryptocurrency is using mobx. Source available [here](https://github.com/appditto/blaise_wallet_flutter)

**Comment by ibhavikmakwana** [comment]
[It's all widgets - podcast](https://play.google.com/store/apps/details?id=com.ibhavikmakwana.itsallwidgetspodcast) - It's a Flutter app that lists all the Flutter Podcast from the itsallwidgets that is hosted by the Hillel Coren.

Source code available [here](https://github.com/ibhavikmakwana/itsallwidgets_podcast).

**Comment by jaumard** [comment]
[L.I.S.A.](https://github.com/mylisabox/lisa_flutter) - It's a Home Automation System based on voice recognition available in mobile/desktop (and web ish lool)

**Comment by kaarimtareek** [comment]
i've made simple app with json placeholder api  [simple_app](https://github.com/kaarimtareek/project_app) with mobx ,well structured and well organized (as i hope) .

**Comment by em-ka** [comment]
**Beer diary** - Free app for finding out what beer is on offer in your area and keeping your beer consumption stats. Made for [Android](https://play.google.com/store/apps/details?id=cz.proteus.pivnidenicek) and [iOS](https://apps.apple.com/cz/app/pivni-denicek/id577774129). 
Thank you all, MobX.dart makes Flutter state-management painless.

**Comment by andredealmei** [comment]
[SISLoja](https://play.google.com/store/apps/details?id=wlssistemas.com.br.sisloja&hl=pt_BR) is a business management application built with [slidy](https://pub.dev/packages/slidy),[modular](https://pub.dev/packages/flutter_modular) and MobX

**Comment by Bryanx** [comment]
I am creating a plant management app [here](https://github.com/Bryanx/fluttery-plants). It has integration with SQLite.  At the moment trying to find out how to integrate authentication and firebase in this.

**Comment by ibhavikmakwana** [comment]
I have been working on the boilerplate [project](https://github.com/ibhavikmakwana/flutter_mobx_boilerplate) includes MobX as a primary state management.
Right now has a below features:

    Init application flow.
    Dummy Login flow with validations,
    Fetching data from internet using http,
    Changing theme (Dark/Light/System)

**Comment by dukaric1991** [comment]
[**Tennboard**](https://www.tennboard.com) App for tennis players to create their own clubs and invite friends :)

**Comment by tomwyr** [comment]
[Pinger](http://github.com/tomwyr/pinger) - ping command utility app.

**Comment by polilluminato** [comment]
[Wassword - Wonderful Password Generator](https://play.google.com/store/apps/details?id=com.albertobonacina.wassword&hl=it): you can easily generate complex passwords and use them for your online accounts. Choose length and chars to be used and generate your passwords safely. Everything works offline and nothing is shared, your safety comes first.

I've got also a GitHub repo: [wassword-flutter](https://github.com/polilluminato/wassword-flutter).

**Comment by leonardocustodio** [comment]
[picPics - Photo Manager](https://picpics.app) uses MobX to manage its states. Since there were too many things happening in parallel MobX helped to organize the code as well as avoiding rebuilding the widget tree too many times. Improving the app performance a lot.

**Comment by dvd-bnc** [comment]
[Leaflet](https://github.com/PotatoProject/Leaflet), a full fledged notes app, for android, ios, macos, windows and linux. Uses mobx to manage preferences, app data, device info and similar stuff.

**Comment by rootasjey** [comment]
![img](https://play-lh.googleusercontent.com/jmzspB_sxHNpGwkGV3u1JPK5tUuP-ZHzU-21FGlqpKa1hTadlB6ZRFlTqW1JQM90c6E=s26)  [fig.style](https://fig.style) is a quotes app & service available on iOS, Android & web. A [REST API](https://dev.fig.style) is on its way ([code source](https://github.com/rootasjey/fig.style)).

**Comment by fzyzcjy** [comment]
My app (to be released officially later) is using it. It has more than 100k lines of Flutter code, with complex logic and user interactions. Mobx is very helpful when building this app!

**Reply by altynbek132** [reply]
Hello! is it already in release?

**Reply by fzyzcjy** [reply]
@altynbek132 Not yet publicly released
